operator/pkg/awsprovider/session.go

```diff
@@ -24,8 +24,10 @@ import (
 	"github.com/aws/aws-sdk-go/aws/request"
 	"github.com/aws/aws-sdk-go/aws/session"
 	"github.com/aws/aws-sdk-go/service/ec2"
-	"github.com/awslabs/karpenter/pkg/utils/log"
-	"github.com/awslabs/karpenter/pkg/utils/project"
+)
+
+const (
+	userAgent = "operator-kit.sh"
```

Comment:
"kubernetes-iteration-toolkit/operator"

Reply:
I don't expect we'll ever use a .sh domain.

Reply:
Thanks, updated.